geo/src/algorithm/voronoi_diagram.rs

```diff
@@ -0,0 +1,970 @@
+use std::{fmt, result};
+
+use crate::{coord, Coord, GeoFloat, Line, MultiPoint, Point, Polygon, Rect, Triangle};
+
+use crate::triangulate_delaunay::{
+    DelaunayTriangle, DelaunayTriangulationError, DEFAULT_SUPER_TRIANGLE_EXPANSION,
+};
+use crate::{BoundingRect, TriangulateDelaunay};
+
+type Result<T> = result::Result<T, VoronoiDiagramError>;
+
+#[derive(Debug, Clone)]
+pub struct VoronoiComponents<T: GeoFloat> {
```

Comment:
undocumented pub struct - pub members should also be documented, or `pub` removed if not needed externally

Reply:
I'm going to add the docs as part of the prep for upstream work.